spec/author_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../classes/author'
```

Comment:
Group Members!

- [ ] You need to write tests for the Items class. Your tests need to cover your `can_be_archived?()`, and `move_to_archive()` methods. Kindly add tests for these methods as a group (you can decide who is going to write tests for them)

> Tests should cover for Item class:
> 
> can_be_archived?(): 
> * should return true if published_date is older than 10 years
> * otherwise, it should return false
> 
> move_to_archive(): 
> * should reuse can_be_archived?() method
> * should change the archived property to true if the result of the can_be_archived?() method is true
> * should do nothing if the result of the can_be_archived?() method is false